CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
-### 0.8.6 (Next)
+### 0.8.7 (Next)
```

Comment:
Let's increment the version to 0.9.0 since this is a new feature as part of this PR.

Reply:
Will do